src/web-animations-next-animation.js

```diff
@@ -44,8 +59,21 @@
         this._animation = scope.newUnderlyingAnimationForGroup(this.effect);
         scope.bindAnimationForGroup(this);
       }
-
-      // FIXME: move existing currentTime/startTime/playState to new animation
+      if (hasUnderlying) {
+        if (oldPlaybackRate != 1) {
+          this.playbackRate = oldPlaybackRate;
+        }
+        if (oldPaused) {
+          this.pause();
+        }
+        if (oldStartTime !== null) {
+          this.startTime = oldStartTime;
+        } else if (oldCurrentTime !== null) {
+          this.currentTime = oldCurrentTime;
```

Comment:
We should prefer setting the current time over the start time.

Otherwise after `a.play(); a.startTime = x;` the currentTime will still be pending